src/transformer/utils/get_multichoice_definition.php

```diff
@@ -18,36 +18,42 @@
 defined('MOODLE_INTERNAL') || die();
 
 function get_multichoice_definition(array $config, \stdClass $questionattempt, \stdClass $question, $lang) {
+    // For some reason, newlines and &nbsp is being added to strings,
+    // in order to remove new lines we have to ensure nbsp is also removed.
+    $replacestrings = array("\n", "&nbsp");
```

Comment:
For consistency with other parts of the code can we change this to the square brackets syntax? This has been done in a few other places too.
```suggestion
    $replacestrings = ["\n", "&nbsp"];
```